tests/unit/utils/test_utils_conferences.py

```diff
@@ -120,3 +120,34 @@ def test_render_contributions():
     result = render_contributions(hits)
 
     assert expected == result
+
+
+def test_render_contributions_handles_unicode():
+    hits = Response({
+        'hits': {
+            'hits': [
+                {
+                    '_type': 'hep',
+                    '_source': {
+                        'control_number': 1427573,
+                        'titles': [
+                            {'title': u'Storage Ring Based EDM Search — Achievements and Goals'},
+                        ],
+                    },
+                },
+            ],
+            'total': 1,
+        },
+    }).hits
+
+    expected = ([
+        [
+            u"<a href='/literature/1427573'>Storage Ring Based EDM Search — Achievements and Goals</a>",
+            u'\n  \n  \n\n\n  \n',
```

Comment:
My point is, there's no point sweeping some ugly output under the rug to pretend it doesn't exist, but we should take it as an occasion to ["listen to our tests"](http://chimera.labs.oreilly.com/books/1234000000754/ch19.html#_listen_to_your_tests_ugly_tests_signal_a_need_to_refactor): this "API" function that returns formatted HTML is bad, and should be refactored.

Besides, a fix for this is already coming in https://github.com/inspirehep/inspire-next/pull/1785/commits/0b90eac5baa560c34b6ece0cef23dbdb2ca6ddce.